corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -213,6 +213,14 @@
       </style>
     {% endif %}
 
+    {% if request.couch_user.is_staff %}
```

Comment:
just confirming that this would not fail if there is no `couch_user` on request? like a non-logged in page.

Reply:
Since the button that shows this modal is only viewable for logged in users, there should always be a `couch_user`.

Reply:
okay, I assume the modal template will always be there on the page, irrespective of its usage since its in the base template.
Can you confirm that this was tested for a non logged in user?

Reply:
I can confirm this works for a non-logged in user as well. The bug report uses a similar approach with the `{% if allow_report_an_issue %}` where `allow_report_an_issue` is a permission check on the couch user.